homeassistant/components/lastfm/config_flow.py

```diff
@@ -0,0 +1,66 @@
+"""Config flow for LastFm."""
+from typing import Any
+
+from pylast import LastFMNetwork, WSError
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.const import CONF_API_KEY
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers.selector import SelectSelector, SelectSelectorConfig
+from homeassistant.helpers.typing import ConfigType
+
+from .const import CONF_USERS, DOMAIN, LOGGER, PLACEHOLDERS
+
+CONFIG_SCHEMA: vol.Schema = vol.Schema(
+    {
+        vol.Required(CONF_API_KEY): str,
+        vol.Required(CONF_USERS): SelectSelector(
```

Comment:
Could put this in a second step so we can populate the list?

Reply:
Populate the list with what? We can't fetch all lastfm users :p

Reply:
No idea how lastfm works but perhaps you already have a "friend-list" or in minimum your own account it can be pre-populated with?

Reply:
Yes you can be friends with people. Hmm, I was thinking that it would block adding people who aren't your friends, but now thinking about it, it's possible to add custom_values, so that would solve it